src/core/utils/components/Popover.tsx

```diff
@@ -65,7 +65,10 @@ export const Popover = React.forwardRef((props: PopoverProps, ref) => {
     interactive: true,
     popperOptions: {
       strategy: 'fixed',
-      modifiers: [{ name: 'flip' }],
+      modifiers: [
+        { name: 'flip' },
+        { name: 'preventOverflow', options: { padding: 0 } },
+      ],
     },
```

Comment:
I just realized that if user adds `popperOptions`, these will all get overwritten. Do we want to merge user's popperOptions here?

Reply:
I think so, we are doing the same with `plugins`. So I added merging.